tests/lwip_sock_*: compilation errors

Compilation of these tests throw the following error message on the ESP8266 platform:
```
.../tests/lwip_sock_ip/stack.c:355:36: error: 'msg.content.value' is used uninitialized in this function [-Werror=uninitialized]
     while (data_len != (msg.content.value - sizeof(ipv6_hdr_t)))
```
which seems to be indeed the case if stack variables are not initialized by default. If I understand the semantic of the source correctly, the following construct could solve the problem:
```
do {
    msg_receive(&msg);
} while (data_len != (msg.content.value - sizeof(ipv6_hdr_t)));

--- a/tests/lwip_sock_ip/stack.c
+++ b/tests/lwip_sock_ip/stack.c
@@ -321,7 +321,7 @@ bool _check_4packet(uint32_t src, uint32_t dst, uint8_t proto,
                     void *data, size_t data_len, uint16_t netif)
 {
 #if LWIP_IPV4
-    msg_t msg;
+    msg_t msg = { .content = { .value = 0 } };
 
     (void)netif;
     while (data_len != (msg.content.value - sizeof(struct ip_hdr))) {
@@ -349,7 +349,7 @@ bool _check_6packet(const ipv6_addr_t *src, const ipv6_addr_t *dst,
                     uint8_t proto, void *data, size_t data_len, uint16_t netif)
 {
 #if LWIP_IPV6
-    msg_t msg;
+    msg_t msg = { .content = { .value = 0 } };
 
     (void)netif;
     while (data_len != (msg.content.value - sizeof(ipv6_hdr_t))) {

--- a/tests/lwip_sock_udp/stack.c
+++ b/tests/lwip_sock_udp/stack.c
@@ -351,7 +351,7 @@ bool _check_4packet(uint32_t src, uint32_t dst, uint16_t src_port,
                     uint16_t netif, bool random_src_port)
 {
 #if LWIP_IPV4
-    msg_t msg;
+    msg_t msg = { .content = { .value = 0 } };
 
     (void)netif;
     while (data_len != (msg.content.value - sizeof(struct ip_hdr))) {
@@ -385,7 +385,7 @@ bool _check_6packet(const ipv6_addr_t *src, const ipv6_addr_t *dst,
                     bool random_src_port)
 {
 #if LWIP_IPV6
-    msg_t msg;
+    msg_t msg = { .content = { .value = 0 } };
 
     (void)netif;
     while (data_len != (msg.content.value - sizeof(ipv6_hdr_t))) {

